test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ControllerTest.cs

```diff
@@ -1771,6 +1771,22 @@ public void TryValidateModelWithInvalidModelNoPrefix_ReturnsFalse()
         }
 
         [Fact]
+        public void TryValidateModel_Succeeds_WithoutValidatorMetadata()
+        {
+            // Arrange
+            var model = new TryValidateModelModel();
+            var binder = new GenericModelBinder();
```

Comment:
Why this model binder in particular? I'd expect model binding to do nothing / not be called.